resources/lrsql/config/config.edn

```diff
@@ -22,7 +22,7 @@
                    ;; :db-properties "currentSchema=lrsql"
 
                    ;; Testing Only! Specify the version used with testcontainers
-                   :test-db-version "11.9"}
+                   :test-db-version #or [#env LRSQL_TEST_DB_VERSION "11"]}
```

Comment:
Do we need to document this env var? We do have _some_ documentation touching upon dev-only stuff (e.g. benching), so I'd say it's a valid question.

Reply:
I can put something in the `dev.md` file about it sure. Don't want to put it in the main env var doc because it isn't runtime.

Reply:
https://github.com/yetanalytics/lrsql/pull/288/commits/4d62b4d0ae9baa44323d810bd3133844db9c2409